test/e2e/infinispan/upgrade_operand_test.go

```diff
@@ -78,7 +84,11 @@ func TestOperandUpgrade(t *testing.T) {
 // installed operand, only result in a StatefulSet rolling upgrade
 func TestOperandCVERollingUpgrade(t *testing.T) {
 	defer testKube.CleanNamespaceAndLogOnPanic(t, tutils.Namespace)
-	versionManager := tutils.VersionManager
+	versionManager := tutils.VersionManager()
+
+	if !versionManager.Latest().CVE {
+		t.Skip("Latest release is non-cve, skipping test")
+	}
```

Comment:
If we're going to skip the test, then we can remove the `if !cveOperand.CVE {` check on L95-99.

Reply:
Removed